config/settings.yml

```diff
@@ -144,6 +144,7 @@ evss:
   mock_ppiu: true
   mock_itf: true
   mock_disabilities_form: true
+  mock_rating_info: true
```

Comment:
Just a heads up that this will need to be overridden for staging and prod in the `devops` repo:

- https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/staging-settings.local.yml.j2
- https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/prod-settings.local.yml.j2